src/main/java/net/sf/jabref/exporter/FieldFormatterCleanups.java

```diff
@@ -85,7 +85,7 @@ public int hashCode() {
         return Objects.hash(actions, enabled);
     }
 
-    private static List<FieldFormatterCleanup> parse(String formatterString) {
+    public static List<FieldFormatterCleanup> parse(String formatterString) {
```

Comment:
So you would say to use the constructor with faked `enabled` value rather than make `parse` public?
`for (FieldFormatterCleanup cleanUp : new FieldFormatterCleanups(true, values.get(1))
                        .getConfiguredActions())` instead of https://github.com/JabRef/jabref/pull/1451/files#diff-1f5b83f0638c8e3856ca20333099555fR410